learn.qmd

```diff
@@ -1,13 +1,73 @@
+  start your own journey.
+- **Case studies** are real-life retrospective accounts of real projects
+  delivered by users in the field. (coming soon)
+- **How-to guides** are question-specific recipe-like scripts to show
+  how multiple R packages can interoperate to get a solution.
+
+Please note the development stage of the learning materials. A "pre-alpha" tutorial is still being constructed. An "alpha" tutorial is being taught by the original authors but has not been fully tested.
+
+## Outbreak Analytics Curriculum
+
+These workshops focus on working with outbreak data using peer-reviewed interoperable open-source software like R packages to help the decision-making process in public health.
+
+### e-Epi training kit en Español
+
+El **e-Epi-training kit** es una estrategia para el aprendizaje en línea que permite un aprendizaje progresivo y flexible. Este kit incluye el curso virtual en ciencia de datos en salud pública y modelamiento de enfermedades infecciosas, el cual está dirigido a América Latina y el Caribe y cuenta con un enfoque de género. [Leer más detalles sobre la estrategia en línea](https://epiverse-trace.github.io/epi-training-kit/).
+
+### Tutoriales en Español
+
+| Tutorial                                           | Sitio                                                                                                                                                    | Repositorio                                                                                        | Agenda para instructoras/es                                                                                                   | Fase de desarrollo                                                                                                                                                   |
+| -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Análisis de Brotes y Modelamiento en Salud Pública | [{{< fa regular window-maximize title="Website">}}](https://epiverse-trace.github.io/epimodelac/) | [{{< fa brands github title="GitHub Repository" >}}](https://github.com/epiverse-trace/epimodelac/) | [{{< fa solid person-chalkboard title="Instructor view">}}](https://epiverse-trace.github.io/epimodelac/instructor/index.html) | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+
```

Comment:
It would be cool to use [these features for tables](https://quarto.org/docs/authoring/tables.html#using-bootstrap-classes) that quarto provides.

```suggestion
: {.striped .hover}

```